core/primitives/src/telemetry.rs

```diff
@@ -37,4 +37,6 @@ pub struct TelemetryInfo {
     pub agent: TelemetryAgentInfo,
     pub system: TelemetrySystemInfo,
     pub chain: TelemetryChainInfo,
+    // Extra telemetry information that will be ignored by the explorer frontend.
```

Comment:
let's mention what's the format here (is it going to be json? or some base64 encoded something?)

Also mention that this is mostly for experimental debugging stuff.